internal/pkg/ccloudv2/flink_gateway.go

```diff
@@ -44,12 +43,12 @@ func (c *FlinkGatewayClient) flinkGatewayApiContext() context.Context {
 
 func (c *FlinkGatewayClient) DeleteStatement(environmentId, statementName, orgId string) error {
 	httpResp, err := c.StatementsSqlV1alpha1Api.DeleteSqlV1alpha1Statement(c.flinkGatewayApiContext(), environmentId, statementName).OrgId(orgId).Execute()
-	return catchFlinkError(err, httpResp)
+	return errors.CatchFlinkError(err, httpResp)
```

Comment:
Do we catch Flink errors in non-Flink commands? Wondering why this function is available globally instead of just inside of this package.

Reply:
Nope, that's only used in flink commands